.github/workflows/docs.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           python-version: "3.10"
       - name: install dependencies
         run: |
-          python -m pip install tox
+          python -m pip install tox>=4
```

Comment:
This was more to prevent more issues like the one with Python 3.6, where `pip install tox` gives you version 3.8. But this was before I found about `tox.min_version`, so I'm fine removing this

Reply:
done!